examples/tracing/tcpv4listen.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env python
```

Comment:
``/usr/bin/env`` makes the script virtualenv proof. I'm not sure the extra PATH lookup is much of an issue given Python and Clang and ... overhead anyway ;)